src/Compilers/CSharp/Test/Emit3/RefReadonlyParameterTests.cs

```diff
@@ -759,27 +759,24 @@ public static void M(ref readonly int x) { }
             }
             """;
         CreateCompilation(source).VerifyDiagnostics(
-            // (4,38): error CS1951: An expression tree lambda may not contain a ref, in or out parameter
-            // Expression<D> e1 = (ref readonly int p) => C.M(in p);
-            Diagnostic(ErrorCode.ERR_ByRefParameterInExpressionTree, "p").WithLocation(4, 38),
-            // (5,52): error CS8329: Cannot use variable 'p' as a ref or out value because it is a readonly variable
-            // Expression<D> e2 = (ref readonly int p) => C.M(ref p);
-            Diagnostic(ErrorCode.ERR_RefReadonlyNotField, "p").WithArguments("variable", "p").WithLocation(5, 52),
-            // (6,38): error CS1951: An expression tree lambda may not contain a ref, in or out parameter
-            // Expression<D> e3 = (ref readonly int p) => C.M(p);
-            Diagnostic(ErrorCode.ERR_ByRefParameterInExpressionTree, "p").WithLocation(6, 38),
-            // (6,48): warning CS9195: Argument 1 should be passed with the 'in' keyword
-            // Expression<D> e3 = (ref readonly int p) => C.M(p);
-            Diagnostic(ErrorCode.WRN_ArgExpectedIn, "p").WithArguments("1").WithLocation(6, 48),
-            // (7,25): error CS1676: Parameter 1 must be declared with the 'ref readonly' keyword
-            // Expression<D> e4 = (int p) => C.M(in p);
-            Diagnostic(ErrorCode.ERR_BadParamRef, "p").WithArguments("1", "ref readonly").WithLocation(7, 25),
-            // (7,28): error CS1661: Cannot convert lambda expression to type 'Expression<D>' because the parameter types do not match the delegate parameter types
-            // Expression<D> e4 = (int p) => C.M(in p);
-            Diagnostic(ErrorCode.ERR_CantConvAnonMethParams, "=>").WithArguments("lambda expression", "System.Linq.Expressions.Expression<D>").WithLocation(7, 28),
-            // (8,49): error CS1615: Argument 1 may not be passed with the 'out' keyword
-            // Expression<Action<int>> e5 = (int p) => C.M(out p);
-            Diagnostic(ErrorCode.ERR_BadArgExtraRef, "p").WithArguments("1", "out").WithLocation(8, 49));
+                // (4,38): error CS1951: An expression tree lambda may not contain a ref, in or out parameter
```

Comment:
Nit: consider reverting the indentation change.

Reply:
done.